Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -48,6 +48,7 @@
 #include "WebExtensionMenuItem.h"
 #include "WebExtensionMessagePort.h"
 #include "WebExtensionPortChannelIdentifier.h"
+#include "WebExtensionSidebarParameters.h"
```

Comment:
should this be guarded?

Reply:
I think it might not be necessary since the contents of the included file are guarded, but I'll go ahead and add one anyways.